core/domain/exp_services.py

```diff
@@ -1275,6 +1650,11 @@ def update_exploration_status_in_search(exp_id):
 
 
 def delete_documents_from_search_index(exploration_ids):
+    """deleting the documents from search index. 
+    Args:
+        exploration_ids: list(str).list of exploration ids of exploration domain object
```

Comment:
List of ids of the explorations whose documents are to be deleted from the search index.